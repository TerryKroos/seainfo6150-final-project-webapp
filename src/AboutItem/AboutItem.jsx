import React from 'react';
import styles from "./AboutItem.module.css";
import ronaldo from "../images/ronaldo.jpg";


const AboutItem = (props)=>{
    return(
      <div className={styles.container}>
         <img src={ronaldo} alt="ronaldopic"/>
         <div className={styles.abouttext}>
            <h4>{props.name}</h4>
            <p className={styles.paragraph}>{props.role}</p>
         </div>
      </div>
      )
}

export default AboutItem;