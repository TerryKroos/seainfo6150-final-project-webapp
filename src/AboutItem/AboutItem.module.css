.container {
    height: 70%;
    width: 100%;
    background-color: #F0F0F0;

}
img {
 width: 100%;   
 height: 100%;

}
.abouttext {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    background-color: #F0F0F0;
    height: 50%;
    width:100%;
}

.paragraph {
    margin-top: -10px;
}
  
