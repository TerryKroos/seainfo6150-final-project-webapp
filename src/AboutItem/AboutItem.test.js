
import React from "react";
import AboutItem from "./AboutItem";
import { render } from "@testing-library/react";

describe("AboutItem tests", () => {
    it("renders correctly", () => {
      const { container } = render(<AboutItem />);
      expect(container).toMatchSnapshot();
    });
  
  });