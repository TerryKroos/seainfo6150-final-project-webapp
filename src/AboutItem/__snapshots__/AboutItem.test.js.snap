// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutItem tests renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <img
      alt="ronaldopic"
      src="ronaldo.jpg"
    />
    <div
      class="abouttext"
    >
      <h4 />
      <p
        class="paragraph"
      />
    </div>
  </div>
</div>
`;
